setup.cfg

```diff
@@ -1,2 +0,0 @@
-[flake8]
-ignore = D211,E731,F821,D401
```

Comment:
I do not think this is safe. It might cause trouble for someone else that has a different set of flake8 subpackages installed. Unless it can be confirmed that `D401` and `D211` was disabled by default in new flake8 version and they have decided to keep it that way.

Reply:
I've tried with an empty `ignore` and there were no issues, but would have to investigate then.